google/gax/api_callable.py

```diff
@@ -155,6 +156,107 @@ def inner(request):
     return inner
 
 
+def _construct_bundling(method_config, method_bundling_override):
+    """Helper for ``construct_settings()``."""
```

Comment:
Although private, this method has sufficient complexity that it warrants documenting the Args and Return value.